### PR TITLE
Bouquet Editor enhancements

### PR DESCRIPTION
**Pull request by wedebe**
This change:

- adds list counts
- adds `Loading...`
- moves `Search` under channel list and updates placeholder text
- de-selects provider when `All channels` selected
- fixes degrees issue (`DVB-S ... 19.2Â°E` -> `DVB-S ... 19.2°E`)

### CODE REVIEW

File: plugin/controllers/i18n.py [hunk 1]
```diff
@@ -432,6 +433,7 @@ def _(txt):
 	'bqe_rename_marker': _("Enter new name for the marker"),
 	'bqe_filename': _("Please enter filename"),
 	'bqe_restore_question': _("Do you really want to restore from file"),
+	'bqe_loading': _("Loading..."),
```
Comment (jbleyel) [inline plugin/controllers/i18n.py:436]: There is a existing string named "Loading" .. I think we need only one.

**Comment by jbleyel** [comment]
And Now for Something Completely Different!

How about?
* move the responsive theme files from `plugin/public/themes/absb` to `plugin/public/absb`
The normal themes and the reponsive can be splitted If we do this.
Many images have splitted owf packages but the old themes are needed If you want the responsive UI.
* merge original bqe and the responsive bqe to one single js to prevent duplicate code
* same for at.js and epgr.js

The merge of the js files will also save a lot of space in flash.

**Comment by wedebe** [comment]
> And Now for Something Completely Different!
> 
> How about?
> 
> * move the responsive theme files from `plugin/public/themes/absb` to `plugin/public/absb`
>   The normal themes and the reponsive can be splitted If we do this.
>   Many images have splitted owf packages but the old themes are needed If you want the responsive UI.
> * merge original bqe and the responsive bqe to one single js to prevent duplicate code
> * same for at.js and epgr.js
> 
> The merge of the js files will also save a lot of space in flash.

You've seen it coming 😁 I've logged an issue at #1083 if you wanna post there and I'll give my feedback!

**Comment by jbleyel** [comment]
absb is the original responsive bootstrap admin template.
One of the VTi team member has used this template.

-> https://github.com/gurayyarar/AdminBSBMaterialDesign

**Comment by wedebe** [comment]
> https://github.com/gurayyarar/AdminBSBMaterialDesign

Ah ok, thanks @jbleyel ! Would there be any objection to renaming it `responsive`?

**Pull request merged**